newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -252,6 +252,42 @@ export const sendAssetAddedToProject = ({
   client.recordEvent('asset-added-to-project', { id, name });
 };
 
+export const sendExtensionDetailsOpened = (name: string) => {
+  if (isDev || !client) return;
+
+  client.recordEvent('extension-details-opened', { name });
+};
+
+export const sendExtensionAddedToProject = (name: string) => {
+  if (isDev || !client) return;
+
+  client.recordEvent('extension-added-to-project', { name });
+};
+
+export const sendExampleDetailsOpened = ({
+  id,
+  name,
+}: {|
+  id: string,
+  name: string,
+|}) => {
+  if (isDev || !client) return;
+
+  client.recordEvent('example-details-opened', { id, name });
+};
+
+export const sendExampleChosenAsProject = ({
```

Comment:
I think this one is not needed because there is `sendNewGameCreated` already giving this info?